packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -299,3 +299,89 @@ class LocalEngineArtifacts extends Artifacts {
     throw Exception('Unsupported platform $platform.');
   }
 }
+
+/// A fuchsia host specific implementation of [Artifacts]
```

Comment:
Remove "host" - technically it's still running on a Linux or Max host, just in a Fuchsia build environment.

(can also remove the "host" from class names and variables below)